python/ribasim_api/tests/test_bmi.py

```diff
@@ -12,12 +12,28 @@ def test_initialize(libribasim, basic, tmp_path):
     libribasim.initialize(config_file)
 
 
+def test_get_start_time(libribasim, basic, tmp_path):
+    basic.write(tmp_path)
+    config_file = str(tmp_path / f"{basic.modelname}.toml")
+    libribasim.initialize(config_file)
+    time = libribasim.get_start_time()
+    assert time == pytest.approx(0.0)
```

Comment:
```suggestion
    assert time == 0.0
```

Wouldn't any slight deviations here be unexpected? This 0 is not the result of any floating point calculations.

Reply:
I am unsure about best practises for these cases.
My feeling is that it's fine to compare floats like `abs(a - b) < eps` for all tests.

But I am fine with changing it to your suggestion if you prefer that.
It is arguably more precise.